curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/667d1bb875f5961913176069.md

```diff
@@ -0,0 +1,149 @@
+---
+id: 667d1bb875f5961913176069
+title: Step 44
+challengeType: 0
+dashedName: step-44
+---
+
+# --description--
+
+Notice how the text `8g` appears centered in the preview.
+
+Nest the `Total Fat` `span` element and the text `8g` in an additional `span` element for alignment.
+
+# --hints--
+
+Nest the `span` element with the class of `bold` and text `Total Fat` and the text `8g`.
+
+```js
+assert(document.querySelector('.daily-value.small-text')?.lastElementChild?.querySelector('span > span')?.innerText === 'Total Fat');
```

Comment:
```suggestion
assert.match(document.querySelector('.daily-value.small-text')?.lastElementChild?.querySelector('span')?.innerHTML, /^<span class=("|')bold\1>Total Fat<\/span> 8g\s?$/);
```

This test wasn't work quite right. It was passing by only checking Total Fat `span`. I came up with this.